### PR TITLE
Phantom types: backgroundPosition

### PR DESCRIPTION
**Pull request by BrianHicks**
progress on the background stuff in #392.

(P.S. not all the ones you've marked me as having done there are done yet. This adds `background-position`, two more PRs will add `background-repeat` and `background-size`.)

### CODE REVIEW

File: src/Css.elm [hunk 1]
```diff
@@ -3809,8 +4003,8 @@ geometricPrecision =
     textRendering optimizeLegibility
 
 -}
-optimizeLegibility: Value { provides | optimizeLegibility : Supported }
-optimizeLegibility=
+optimizeLegibility : Value { provides | optimizeLegibility : Supported }
+optimizeLegibility =
```
Comment (BrianHicks) [inline src/Css.elm]: whoops, I didn't notice this. It must have come in with elm-format. I can separate it into a separate PR if necessary but it seems silly for such a small fix.

Comment (BrianHicks) [reply src/Css.elm]: also kinda ironic that it happened on this particular function 😆

Comment (rtfeldman) [reply src/Css.elm]: it's now fixed on the branch if you want to rebase!

Comment (BrianHicks) [reply src/Css.elm]: done 👍

File: src/Css.elm [hunk 2]
```diff
@@ -3446,6 +3449,196 @@ backgroundImages (Value first) rest =
...
+
+`backgroundPosition` sets the horizontal direction. If you need the vertical
+direction instead, use [`backgroundPosition2`](#backgroundPosition2) like this:
+
+    backgroundPosition zero (px 45)
+
+If you need to set the offsets from the right or bottom, use
+[`backgroundPosition4`](#backgroundPosition4) like this:
+
+    backgroundPosition4 right_ (px 20) bottom_ (pct 25)
```
Comment (rtfeldman) [inline src/Css.elm]: Nice touch! :heart_eyes:

**Comment by rtfeldman** [comment]
Fantastic! Thanks @BrianHicks! :tada:

**Pull request merged**